### DISCUSSION TITLE
Moving Giscus critical variables to .env

### DISCUSSION THREAD

**Discussion by Music47ell** [timlrx/tailwind-nextjs-starter-blog#140] [Ideas]
I think we should move Giscus critical variables to a `.env` file to prevent unwanted comments like [this one](https://github.com/timlrx/tailwind-nextjs-starter-blog/discussions/138).

What I think happened is that someone cloned or forked this repository and tested the comment system without changing the settings inside https://github.com/timlrx/tailwind-nextjs-starter-blog/blob/master/data/siteMetadata.js causing their comment to appear in this template Discussion tab.

I propose that we do the following:

1. Move the following Giscus variables to a `.env` file:  `repo, repositoryId, category, categoryId`
2. Create a `.env.example` file similar to this one https://github.com/leerob/leerob.io/blob/main/.env.example and add the appropriate variables.

This template is gaining a lot of attention and is increasing in popularity. Keeping the variables the way they are right now could cause more unwanted comments to appear and even spam.

I tested adding a `.env` file locally and it works great. I can send a PR soon if you agree with my proposal.

**Comment by timlrx** [comment]
Yes, let's do that.  Noticed the unwanted comment as well 😆

**Reply by Music47ell** [reply]
PR https://github.com/timlrx/tailwind-nextjs-starter-blog/pull/141 is ready for your approval.
